devops/demo/landing-page/nginx.conf

```diff
@@ -23,6 +23,7 @@ http {
         location / {
             root /opt/nginx/webroot;
             index index.html;
+            add_header Content-Security-Policy "default-src 'self'; img-src 'self' https://securedrop.org; object-src 'none'; frame-ancestors 'none';" always;
```

Comment:
I would suggest having default-src be 'none' and then explicitly setting script-src, style-src, font-src and img-src. That ways we don't need to set other things like object-src, and frame-ancestors to none.

Reply:
Makes sense, done.